client/src/Components/CommonComponents/AutoCompleteTags/index.jsx

```diff
@@ -18,7 +18,8 @@ export default class AutoCompleteTags extends Component {
   componentDidMount() {
     // git the data form the back for skills and offer_type and setSatet for the skills and offer type
     // const selected = options.filter(item => _.find(fetched, id => id === item.id))
-    const { type } = this.props;
+    const { type, allowNew } = this.props;
+    this.setState({ allownew: allowNew });
```

Comment:
@Fatmasiam I think you don't need to put `allowNew` into state, because you already have it in props.